page.codeberg.ilijaculap.snux.yml

```diff
@@ -0,0 +1,90 @@
+app-id: page.codeberg.ilijaculap.snux
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: "/app/run-snux.sh"
+modules:
+- name: tkinter
+  buildsystem: simple
+  build-commands:
+    - pip3 install --prefix=${FLATPAK_DEST} .
+  sources:
+    - type: git
+      url: https://github.com/iwalton3/tkinter-standalone
+      commit: d9cb97c5bd4f814c73678366e0e48220776b6ad3
+ 
+  modules:
+  - name: tcl
+    buildsystem: autotools
+    subdir: unix
+    post-install:
+      - chmod 755 /app/lib/libtcl*.so
+    cleanup:
+      - /bin
+      - /lib/pkgconfig
+      - /man
+    sources:
+      - type: archive
+        url: https://prdownloads.sourceforge.net/tcl/tcl8.6.13-src.tar.gz
+        sha256: 43a1fae7412f61ff11de2cfd05d28cfc3a73762f354a417c62370a54e2caf066
+        x-checker-data:
+          type: anitya
+          project-id: 4941
+          stable-only: true
+          url-template: https://prdownloads.sourceforge.net/tcl/tcl$version-src.tar.gz
+  
+  - name: tk
+    buildsystem: autotools
+    subdir: unix
+    post-install:
+      - chmod 755 /app/lib/libtk*.so
+    cleanup:
+      - /bin
+      - /lib/pkgconfig
+      - /man
+    sources:
+      - type: archive
+        url: https://prdownloads.sourceforge.net/tcl/tk8.6.13-src.tar.gz
+        sha256: 2e65fa069a23365440a3c56c556b8673b5e32a283800d8d9b257e3f584ce0675
+        x-checker-data:
+          type: anitya
+          project-id: 11426
+          stable-only: true
+          url-template: https://prdownloads.sourceforge.net/tcl/tk$version-src.tar.gz
+
+- name: python3-pillow
+  buildsystem: simple
+  build-commands:
+    - python3 -m pip install --prefix=${FLATPAK_DEST} .
+  build-options:
+    build-args:
+      - --share=network
```

Comment:
this is NOT allowed. build won't work. Use the flatpak-builder-tools to generate a manifest.